packages/sdk/contractkit/src/wrappers/Attestations.ts

```diff
@@ -234,30 +233,6 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
     (stat) => ({ completed: valueToInt(stat[0]), total: valueToInt(stat[1]) })
   )
 
-  /**
-   * Returns the verified status of an identifier/account pair indicating whether the attestation
-   * stats for a given pair are completed beyond a certain threshold of confidence (aka "verified")
-   * @param identifier Attestation identifier (e.g. phone hash)
-   * @param account Address of the account
-   * @param numAttestationsRequired Optional number of attestations required.  Will default to
-   *  hardcoded value if absent.
-   * @param attestationThreshold Optional threshold for fraction attestations completed. Will
-   *  default to hardcoded value if absent.
-   */
-  async getVerifiedStatus(
```

Comment:
Maybe we actually want to keep this one since it's a view function? Downside would be adding back the `isAccountConsideredVerified` logic here or else keeping that function :-/

Reply:
added back!